containers/kubernetes/scripts/install-rpms.sh

```diff
@@ -30,7 +30,12 @@ while [ $# -gt 0 ]; do
 done
 
 if [ -z "$COCKPIT_RPM_URL" ]; then
-    COCKPIT_RPM_URL="https://kojipkgs.fedoraproject.org/packages/cockpit"
+    if [ $arch == "ppc64le" ]
```

Comment:
@stefwalter @petervo now updating COCKPIT_RPM_URL by default in the dockerfile, PTAL into latest commit.